docs/rules.md

```diff
@@ -211,6 +211,8 @@ Options:
 	- `single-line space delimited`: ```tags: [tag1 tag2]```
 	- `single string space delimited`: ```tags: tag1 tag2```
 	- `single string comma delimited`: ```tags: tag1, tag2```
+- Format yaml tags section: Turns on formatting for the yaml tags section.
```

Comment:
This file is auto generated from rules by docs.ts, you probably want to make this change somewhere else.

Reply:
Yes, this file is indeed autogenerated. If it is not meant to be committed, then I suggest this file to be added to the `.gitignore`

Reply:
@mnaoumov , the changes listed here were created by running `doc.js`  via the npm commands, right? If so, then we are good to have this merged.

I just want to clear up any questions around how this change was created.

Reply:
Oh sorry, confused myself. Yep, I think we should keep it this way, I don't see another way to host the docs.